sql/report_queries/title_count_brief/README.md

```diff
@@ -1,2 +1,5 @@
-# ACRL Title Count Report
+This report is superseded by the [ACRL Title Count query](https://github.com/NancyBolduc/folio-analytics/tree/master/sql/report_queries/title_count). It was created in the early stage of FOLIO report writing and is no longer officially used for the ACRL Title Count report. 
```

Comment:
We should be careful with absolute links. As links are supposed to work in different branches and repositories Id' suggest to change it to  `../title_count`

Reply:
I made the change Axel. Thanks for the learning opportunity!